### ISSUE TITLE
FNX-5441 ⁃ Native Dark Theme

### ISSUE THREAD

**Issue by yoasif** [mozilla-mobile/fenix#208]
As a user, I want to select a browser theme that is dark so that it can match my default dark Android theme or differ from my default light theme.

_Note - this feature is no longer part of Battery Saving and will need UX spec for settings_

UX issue #826

### Acceptance Criteria
I can allow Fenix to display the theme that matches my default Android theme
I will find enough visual contrast on both light and dark themes to meet a11y standards
I can change themes between dark and light

### Why/User Benefit/User Problem
Many users would like a dark mode across their device, and Android now has a setting for this. This ideally would also apply to versions prior to Pie as well, as a setting option. See  #209.

Android setting: https://www.cnet.com/how-to/how-to-enable-android-pies-new-dark-mode-and-save-battery-life-p/

Dark mode also has impacts on battery life (less power used): https://www.cnet.com/news/using-androids-dark-mode-improves-battery-life-google-confirms-p/

**Comment by owendaprile** [comment]
A dark mode should not only be pure black. There’s a lot of evidence out there against pure black backgrounds (https://uxmovement.com/content/why-you-should-never-use-pure-black-for-text-or-backgrounds/). OLED screens also benefit linearly in power reduction from white to black, so a gray background saves a lot of battery anyway. Any dark mode should be gray by default with pure black as an option.

**Comment by digitalcircuit** [comment]
I'd like to reaffirm support for "pure black as an option" - some people use pure black for accessibility reasons, e.g. hypersensitivity to light.  The linked article from @owendaprile mentions this under `High Contrast for Accessibility`.

Alternatively, Fenix could implement a dark theme in a way that's similar to the approach for desktop Firefox - open up every color to themes, so pure black could be a third-party addition.

**Comment by sprite-1** [comment]
I agree with the dark mode not be pure black by default. Some apps only offer that as their "dark mode" and the contrast is too much for me personally

**Comment by bbinto** [comment]
As per [comment](https://github.com/mozilla-mobile/fenix/issues/240#issuecomment-473414507), we might pull this back into a Battery saving story after MVP, however it should land for MVP as an individual feature.

**Comment by bsurd** [comment]
This has been tested and verified by QA, removing the QA needed label but leaving the issue open until all blocking/related issues are fixed.

**Comment by bifleming** [comment]
@ekager

**Comment by kbrosnan** [comment]
Blocking issue has been closed.
